### PR TITLE
Exclude COINSWITCH from smartWallet exchange offers

### PR DESCRIPTION
**Pull request by alazarevski**
exclude COINSWITCH from smartWallet exchange offers

### CODE REVIEW

File: src/actions/exchangeActions.js [hunk 1]
```diff
@@ -255,7 +256,7 @@ export const searchOffersAction = (fromAssetCode: string, toAssetCode: string, f
       connectExchangeService(getState());
       exchangeService.onOffers(offers =>
         offers
-          .filter(({ askRate }) => !!askRate)
+          .filter(({ askRate, provider }) => !!askRate && !(isSmartWallet && provider === 'COINSWITCH-SHIM'))
```
Comment (jsidorenko) [inline src/actions/exchangeActions.js:259]: How about creating a variable for providers to filter out?

Comment (jsidorenko) [reply src/actions/exchangeActions.js:259]:
```
const excludeProviders = [];
if (isSmartWallet) {
  excludeProviders.push('COINSWITCH-SHIM');
}
...
offers.filter(({ askRate, provider }) => !!askRate && !excludeProviders.includes(provider))
```

File: src/actions/exchangeActions.js [hunk 2]
```diff
@@ -188,6 +188,7 @@ export const searchOffersAction = (fromAssetCode: string, toAssetCode: string, f
     } = getState();
 
     const activeWalletId = getPreferredWalletId(accounts);
+    const isSmartWallet = isSmartWalletActive(accounts);
```
Comment (jsidorenko) [inline src/actions/exchangeActions.js:191]: You can get the same without creating new util: `const isSmartWallet = isActiveAccountSmartWalletSelector(getState())`

Comment (alazarevski) [reply src/actions/exchangeActions.js:191]: too much passing around

File: src/utils/smartWallet.js [hunk 3]
```diff
@@ -117,6 +117,11 @@ export const getSmartWalletStatus = (
   };
 };
 
+export const isSmartWalletActive = (accounts: Accounts) => {
+  const activeAccount = getActiveAccount(accounts);
+  return !!activeAccount && activeAccount.type === ACCOUNT_TYPES.SMART_WALLET;
```
Comment (jsidorenko) [inline src/utils/smartWallet.js:122]:
```suggestion
  return !!activeAccount && checkIfSmartWalletAccount(activeAccount);
```

File: src/actions/exchangeActions.js (hunk 2 above)
Comment (jsidorenko) [reply src/actions/exchangeActions.js:191]: This is what are selectors for

File: src/configs/exchangeConfig.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+export const EXCLUDED_SMARTWALLET_PROVIDERS = [
+  'COINSWITCH-SHIM',
+];
```
Comment (IAmKio) [inline src/configs/exchangeConfig.js:23]: Just a note: anything like this will be moved to remote configuration in the near future.

**Pull request merged**